final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,174 @@
+import requests
+import argparse
+import json
+import logging
+import socket
+import bs4
+from bs4 import BeautifulSoup
+
+
+log = logging.getLogger("Log")
+
+
+class CustomEx(Exception):
+    def __init__(self, msg):
+        self.msg = msg
+        super().__init__(msg)
```

Comment:
It is better to name if after the situation you are going to raise it, for example `ConnectionError` and etc.
Also it is better to have separate module for it.
And also you can just implement it like this:
```
class CustomEx(Exception):
    pass
```

That will work practically the same way you implemented, but without redundant code.

Reply:
Can I have a single module for all custom exceptions?